src/views/annual-report/2020/annual-report.scss

```diff
@@ -443,7 +447,7 @@ a, a:link, a:visited, a:active{
     }
 
     .four-ps-peers {
-        background-image: url("/images/annual-report/2020/mission-vision/Peers Splash.svg");
+        background-image: url("/images/annual-report/2020/mission-vision/Peers-splash.svg");
```

Comment:
could we change the filenames for the 4Ps so that they all have the same capitalization convention?